src/client/app/services/request.ts

```diff
@@ -19,7 +19,11 @@ export class Request {
         return this._response(xhr, responseType);
     }
 
-    get<T>(url: string, responseType: ResponseTypes, headers?: Object): Observable<T> {
+    get<T>(url: string, responseType: ResponseTypes, headers?: Object, useCache?: boolean): Observable<T> {
```

Comment:
generally an optional boolean parameter should be something where the default is `false`.  Here you have it as true.
Could you reverse the naming/functionality such that it's more of a `forceBypassCache`?